src/raymath.h

```diff
@@ -469,7 +469,7 @@ RMAPI Vector2 Vector2Rotate(Vector2 v, float angle)
 }
 
 // Move Vector towards target
-RMAPI Vector2 Vector2MoveTowards(Vector2 v, Vector2 target, float maxDistance)
+RMAPI Vector2 Vector2MoveTowards(Vector2 v, Vector2 target, float speed)
```

Comment:
Hello,

I'm assuming this was submited by mistake, because the only thing this does is break `Vector2MoveTowards`.

Best regards.